code/more.html

```diff
@@ -0,0 +1,68 @@
+<!DOCTYPE html>
+<html>
+
+  <head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Project Name</title>
+    <link rel="stylesheet" type="text/css" href="style.css">
+    <!-- If you're using google fonts, that should be added here -->
+  </head>
+  <body>
+    <!--Menu-->
+    <menu>  
+      <div class="menu-left">
+        <a href="./index.html">Nikki Böhler</a>
+      </div>
+      <div class="menu-right">
+        <a href="./more.html">More</a>
+        <a href="./contact.html">Contact</a>
+      </div>
+    </menu>
+    
+    <!-- Title-->
+    <h1>What I'm up to</h1>
+
+    <!-- Description-->
+    <div>
+    </div>
+
+    <!-- Current Roles-->
+    <div class="boxes">
+      <h2>Current Roles</h2>
+      <ul>
+        <li>Opendata.ch, Director</li>
+        <li>Arui, Co-Director</li>
+        <li>SGG, Board Member</li>
+      </ul>
+    </div>
+
+     <!-- Ventures I co-founded-->
+     <div class="boxes">
+      <h2>Ventures I Co-Founded</h2>
+      <ul>
+        <li>Prototype Fund (Incubation Program)</li>
+        <li>Digitales Lernlabor (NGO)</li>
+        <li>Data Café (Awareness Campaign)</li>
+      </ul>    
+    </div>
+
+    <!-- Public-->
+    <div class="boxes">
+      <h2>Media, Publications & Talks</h2>
+      <ul>
+        <li>«Die Förderung von Datenkompetenz ist auch Aufgabe der Medien» – Article on data literacy in SRG Link Magazine</li>
+        <li>My recommendations at the onset of the pandemic – Opinion statement in Strategiedialog 21</li>
+        <li>Warum ein Schweizer Verein für mehr Klimadaten kämpft – Interview about climate data in Yova’s Blog </li>
+      </ul>     
+    </div>
```

Comment:
Maybe we can use the section tag for all of this boxes instead of divs to make them more semantic?